pkgs/tools/misc/cod/default.nix

```diff
@@ -15,7 +15,16 @@ buildGoModule rec {
 
   ldflags = [ "-s" "-w" "-X main.GitSha=${src.rev}" ];
 
-  doCheck = false;
+  nativeBuildInputs = [ python3 ];
```

Comment:
If it's a `checkPhase`-only dependency, better to add it to `checkInputs` because semantics.